lxd/storage/drivers/driver_dir.go

```diff
@@ -513,6 +514,7 @@ func (d *dir) RenameVolume(volType VolumeType, volName string, newVolName string
 		return err
 	}
 
+	// tomp TODO check old snapshots dir is removed.
```

Comment:
@stgraber I'm aware of this, just want to make sure that the old volume's snapshot dir is removed.